storage/chat.go

```diff
@@ -77,6 +77,37 @@ func (m *MongoStorer) GetTopMessages(top int) (result TopMessages, err error) {
 	return
 }
 
+func (m *MongoStorer) GetUserMessageCount(discordUserID string) (messageCount int, err error) {
+	messageCount, err = m.chat.Find(bson.M{"discorduserid": discordUserID}).Count()
```

Comment:
Curious if this'll be slow for people with many messages, but if that's the case we'll have to try a new pull request then